docs/running-tests/from-local-system.md

```diff
@@ -27,38 +43,36 @@ pip install virtualenv
 ```
 
 To get the tests running, you need to set up the test domains in your
```

Comment:
This isn't actually true now in the common case. You only need to do this if you are testing something other than Chrome, Firefox and Servo (and probably other Blink-based browsers would work with the same prefs set we use for Chrome).

Reply:
Oh and it's necessary if you want to use something other than wptrunner e.g. just wpt serve. But there is a hppy path here where you don't need to set this up and we should mention that.

Reply:
The text could look something like this:

    If you expect to [use `wpt serve` directly](#via-the-browser) or test a
    browser other than Firefox or Chrome, then you'll need to set up the test
    domains in your [`hosts`
    file](http://en.wikipedia.org/wiki/Hosts_%28file%29%23Location_in_the_file_system)."

That explanation for the "happy path" is potentially more distracting than just saying, "run this command," even given that it might be unnecessary. My clumsy wording could probably be improved, but readers will still have to skip ahead to learn about the differences of the two test execution methods. More fundamentally, they'll have to remember to revisit the instructions if they later decide to use different features.

Reply:
This is true. Butnot having to alter your system config to run tests is an important usability win in the cases where it applies. For example exactly the same set of caveats apply when running from a gecko checkout. But because by far the most commin requirement is "run tests in gecko using wptrunner", almost no one has to reconfigure their system and the lack of documentation isn't a problem.